Euphorolog.Services/DTOValidators/AuthDTOValidators/SignUpRequestDTOValidator.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Euphorolog.Services.DTOs.AuthDTOs;
+using FluentValidation;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Euphorolog.Services.DTOValidators.AuthDTOValidators
+{
+    public class SignUpRequestDTOValidator : MainDTOValidator<SignUpRequestDTO>
+    {
+
+        public SignUpRequestDTOValidator()
+        {
+            RuleFor(u => u.userName)
+                .NotNull().WithMessage("Username can not be null.")
+                .NotEmpty().WithMessage("Username can not be empty.");
+            RuleFor(u => u.fullName)
+                .NotNull().WithMessage("Fullname can not be null.")
+                .NotEmpty().WithMessage("Fullname can not be empty.");
+            RuleFor(u => u.eMail)
+                .NotNull().WithMessage("Email can not be null.")
```

Comment:
The parameter in `WithMessage()` is what the generic message would say when you define rule `.NotNull()`. So the messages can be considered redundant.